content/tokio/tutorial/bridging.md

```diff
@@ -0,0 +1,384 @@
+---
+title: "Bridging with sync code"
+---
+
+In the examples we have seen so far, we marked the main function with
+`#[tokio::main]` and made the entire project asynchronous. However, this is not
+desirable for all projects. For instance, a GUI application might want to run
+the GUI code on the main thread and run a Tokio runtime next to it on another
+thread.
+
+This page explains how you can isolate async/await to a small part of your
+project.
+
+# What `#[tokio::main]` expands to
+
+The `#[tokio::main]` macro is a macro that replaces your main function with a
+non-async main function that starts a runtime and then calls your code. For
+instance, this:
+```rust
+#[tokio::main]
+async fn main() {
+    println!("Hello world");
+}
+```
+is turned into this:
+```
+fn main() {
+    tokio::runtime::Builder::new_multi_thread()
+        .enable_all()
```

Comment:
I might add a note explaining what `enable_all()` is and a link to the API docs that dig into it more.